visualmetrics/visualmetrics.py

```diff
@@ -2200,6 +2305,25 @@ def main():
         help="Wait for a full white frame after a non-white frame "
         "at the beginning of the video.",
     )
+    parser.add_argument(
+        "--keycolor",
+        action="append",
+        nargs=8,
+        metavar=(
+            "key",
+            "red_low",
+            "red_high",
+            "green_low",
+            "green_high",
+            "blue_low",
+            "blue_high",
+            "fraction",
+        ),
+        help="Identify frames that match the given channel (0-255) low and "
+        "high. Fraction is the percentage of the pixels per channel that "
+        "must be in the given range (0-1).",
+    )
+    parser.add_argument("--keycolors", help="Key color frames output file.")
```

Comment:
I'm not seeing this being used in the browsertime call for visual metrics - can you elaborate on what the purpose of this argument is?

Reply:
Ah ok, you could leave it if you want, but thanks for removing it!